cmd/bundle/plan.go

```diff
@@ -79,13 +80,59 @@ func newPlanCommand() *cobra.Command {
 			return root.ErrAlreadyPrinted
 		}
 
+		// Count actions by type and collect formatted actions
+		createCount := 0
+		updateCount := 0
+		deleteCount := 0
+		changed := make(map[string]bool)
+
+		for _, change := range plan.GetActions() {
+			changed[change.ResourceKey] = true
+			switch change.ActionType.StringShort() {
+			case "create":
+				createCount++
+			case "update", "update_with_id":
+				updateCount++
+			case "delete":
+				deleteCount++
+			case "recreate":
+				// A recreate counts as both a delete and a create
+				deleteCount++
+				createCount++
+			}
+		}
+
+		// Calculate number of all unchanged resources
+		unchanged := 0
+		rv := b.Config.Value().Get("resources")
+		if rv.Kind() != dyn.KindInvalid && rv.Kind() != dyn.KindNil {
```

Comment:
You can use the typed `b.Config.Resources.AllResources()` to make this a bit simpler.